src/test/fuzz/p2p_transport_serialization.cpp

```diff
@@ -21,14 +21,17 @@
 
 namespace {
 
-std::vector<std::string> g_all_messages;
+auto g_all_messages = g_all_net_message_types;
 
 void initialize_p2p_transport_serialization()
 {
     ECC_Start();
     SelectParams(ChainType::REGTEST);
-    g_all_messages = getAllNetMessageTypes();
-    std::sort(g_all_messages.begin(), g_all_messages.end());
+    // Sort, so that changes in the order of elements in g_all_net_message_types
+    // do not cause changes in fuzzing for the same fuzz input.
+    std::sort(g_all_messages.begin(), g_all_messages.end(), [](const char* a, const char* b) {
+        return strcmp(a, b) < 0;
```

Comment:
Just tried that - if I store the constants as `string_view`, e.g. `constexpr std::string_view VERSION{"version"};`, then I get this greeting:

```
test/util/net.cpp:34:9: error: no matching function for call to 'Make'
   34 |         NetMsg::Make(NetMsgType::VERSION,
      |         ^~~~~~~~~~~~
./netmessagemaker.h:14:23: note: candidate function template not viable: no known conversion from 'const std::string_view' (aka 'const basic_string_view<char>') to 'std::string' (aka 'basic_string<char>') for 1st argument
   14 |     CSerializedNetMsg Make(std::string msg_type, Args&&... args)
      |                       ^    ~~~~~~~~~~~~~~~~~~~~
```

Reply:
Then another:

```
net.cpp:3705:31: error: no viable overloaded operator[] for type 'mapMsgTypeSize' (aka 'map<basic_string<char>, unsigned long>')
 3705 |         mapRecvBytesPerMsgType[msg] = 0;
      |         ~~~~~~~~~~~~~~~~~~~~~~^~~~
/usr/include/c++/v1/map:1203:18: note: candidate function not viable: no known conversion from 'const value_type' (aka 'const std::string_view') to 'const key_type' (aka 'const std::string') for 1st argument
 1203 |     mapped_type& operator[](const key_type& __k);
      |                  ^          ~~~~~~~~~~~~~~~~~~~
```

It is possible to get over that by changing `mapMsgTypeSize` and `NET_MESSAGE_TYPE_OTHER` to:
```cpp
inline constexpr std::string_view NET_MESSAGE_TYPE_OTHER{"*other*"};
using mapMsgTypeSize = std::map</* message type */ std::string_view, /* total bytes */ uint64_t>;
```
but then there is another error from `rpc/net.cpp` and I think this would spill off the scope of this PR.

Reply:
Changed to `std::string` and dropped the `std::sort` modification. Thanks!